ert_shared/ensemble_evaluator/prefect_ensemble/storage_driver.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+
+
+def storage_driver_factory(config, run_path):
```

Comment:
Given that this function expects two entries in the config, perhaps that is a bit overboard? Could instead accept a `storage_path` and `type`? Also, in the context of the storage I think `load_path` is both more describing and more honest towards what is passed on as this argument in various places.

Reply:
I agree this should be refactored but I would like to do that in a separate issue also maybe do it after we expand the functionality of the storage driver to also support the not shared disk assumption.